admission-webhook/main.go

```diff
@@ -98,6 +103,19 @@ func env_float(key string, defaultFloat float32) float32 {
 	return defaultFloat
 }
 
+func env_bool(key string) bool {
+	if v, found := os.LookupEnv(key); found {
+		// Convert string to bool
+		if boolValue, err := strconv.ParseBool(v); err == nil {
+			return boolValue
+		}
+		logrus.Warningf("unable to parse environment variable %s with value %s to bool, use default value false", key, v)
```

Comment:
Can we add a test case for this?
Should we error here?

On one hand `env_int` just fails silently and on the other hand `env` will cause a panic.

I think that if the env var is set and we can't parse the value - we should consider throwing error'ing out because the intention was the enable new functionality (RANDOM_HOSTNAME in this case).

Reply:
I will add a test case. I would prefer that throw an error when parsing failure, will change accordingly for env_bool, but would like to keep env_int as it is, not part of this PR

Reply:
sounds good to me